.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,98 @@
+name: 🐞 Bug
+description: File a bug/issue
+# title: "[BUG] <title>"
```

Comment:
Don't leave unnecessary comments please.

Reply:
> Don't leave unnecessary comments please.

It's not yet completed am trying to figure out what I have to add. After the completion I'll let you know then you can review.

Reply:
I didn't complete in finishing the changes you can check after the completion

Reply:
Oh! sorry, made the PR a draft, you can change it back to a normal PR when you are ready.